tests/testthat/test-sas-results-part-1.R

```diff
@@ -0,0 +1,139 @@
+test_that("Posterior summaries for probabilities of
+          DLT (2-parameter logistic model) and recommended doses (NCRMLoss):
+          crmPack vs. SAS", {
+  # Data examples for scenario 1
+  set.seed(0504201914)
+  mcmc_options <- McmcOptions(
+    burnin = 5000,
+    step = 2,
+    samples = 1000000,
+    rng_kind = "Wichmann-Hill",
+    rng_seed = 1
+  )
+
+  dose_grid_sc1 <- c(10, 20, 35, 50, 65, 80, 90, 100)
+  tab_sc1 <- vector("list")
+  tab_sc1$"Data example 1 (DLTs: 0)" <- Data(
```

Comment:
For maintenance and readability, it would actually be better to have each data example in a separate `test_that()` call. Would that be possible?